client_src/js/dashboard/dashboard.worker.js

```diff
@@ -1,103 +0,0 @@
-import currentOperations from "../lib/modules/currentOperations";
-import { DashUpdate } from "./dashboard.updater";
-import UI from "../lib/functions/ui";
-
-const workerResource = "/assets/dist/dashboard-sse.client.min.js";
-
-// Move out to generic worker template
-function handleVisibilityChange() {
-  if (document.hidden) {
-    if (worker !== null) {
-      console.log("Screen abandoned, closing web worker...");
-      worker.terminate();
-      worker = null;
-    }
-  } else {
-    if (worker === null) {
-      console.log("Screen resumed... opening web worker...");
-      createWebWorker();
-    }
-  }
-}
-
-function createWebWorker() {
-  worker = new Worker(workerResource);
-  worker.onmessage = async function (event) {
```

Comment:
I'd like to revert this change. The code was neatly split up in order to see what is asynchronous through SSE and not the same as the rest of the normal sync dashboard code.
I couldve named it dashboard-client.service.js and put it in a backend client service folder or similar to make that idea clearer